packages/paste-website/src/component-examples/KeyboardKeyExamples.ts

```diff
@@ -0,0 +1,191 @@
+export const basicExample = `<KeyboardKeyGroup>
+  <KeyboardKey>Control</KeyboardKey>
+  <KeyboardKey>B</KeyboardKey>
+</KeyboardKeyGroup>
+`.trim();
+
+export const defaultExample = `<>
+    <KeyboardKey>S</KeyboardKey>
+    <Paragraph>Press{" "}
+        <KeyboardKeyGroup>
+            <KeyboardKey>Control</KeyboardKey>
+            <KeyboardKey>B</KeyboardKey>
+        </KeyboardKeyGroup>{" "}to save.
+    </Paragraph>
+</>
+`.trim();
+
+export const disabledExample = `const DisabledExample = () => {
```

Comment:
suggestion: should we be using `aria-keyshortcuts` in any applicable examples here?

Reply:
I added them in and tested but it didn't announce anything due to the MenuItem primitive which was wierd

Reply:
womp womp. at least we'll show people how to do the right thing